<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userEditWindow</class>
 <widget class="QDialog" name="userEditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="id">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="isId">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="isUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>130</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>130</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Script MT</family>
     <pointsize>30</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit User's Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Enter The User's ID or Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Enter The New Information</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>441</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="changeUser">
      <property name="text">
       <string>Change Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newUsername"/>
    </item>
    <item>
     <widget class="QCheckBox" name="changePass">
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPass"/>
    </item>
    <item>
     <widget class="QCheckBox" name="makeAdmin">
      <property name="text">
       <string>Make The User an Admin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="makeRegular">
      <property name="text">
       <string>Make The User a Regular User</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
